develop/parachain-devs/get-started/polkadot-sdk/get-started-psdk.md

```diff
@@ -0,0 +1,102 @@
+  one for Polkadot). Polkadot itself is built with Substrate.
+- [**Cumulus**](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/index.html#cumulus){target=_blank} -
+  A set of libraries and tools which pertain specifically with connecting blockchains to Polkadot,
+  known as parachains.
+
+For an in-depth dive into the monorepo, the [Polkadot SDK Rust documentation](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/index.html){target=_blank} is highly recommended
+
+<!-- TODO: Glossary link ParaId -->
+
+## Zero to Hero with The Polkadot SDK
+
+The process of using the Polkadot SDK to deploying a parachain on a [core](https://wiki.polkadot.network/docs/learn-agile-coretime#core){target=_blank} can be summarized as follows. It is also possible to build standalone blockchains with the Polkadot SDK:
+
+1. **Reserving** a `ParaId`, where the runtime and genesis state will be uploaded
+2. **Compiling** the runtime (written in Rust) to a WebAssembly blob, thereby defining how state transitions from one state to the next. This runtime is created using the Polkadot SDK
+3. **Generating** a chain spec that is viable and ready to be deployed as a live, working parachain
+4. **Generating** the genesis state and wasm
+5. **Obtaining** a core through a [Coretime marketplace](https://wiki.polkadot.network/docs/learn-guides-coretime-marketplaces){target=_blank}
+6. **Assigning** that core to a `ParaId`
+7. **Provisioning** at least one honest, synced collator is available for the task to begin producing blocks
+
+!!!info "What is a task?"
+    A task refers to a process utilizing Polkadot's compute. This could be a parachain or any other computational process, provided that it adheres to the Polkadot protocol. The full definition can be found on the [Polkadot Wiki](https://wiki.polkadot.network/docs/learn-agile-coretime#task).
+
+### Deploying on a Core
+
+Once the desired runtime and pallets have been developed, it will be able to deploy it on a core, which is how one utilizes the shared security of the Polkadot network. One does so by:
+
+```mermaid
+flowchart TD
+    subgraph GA["Generate Artifacts"]
+        direction LR
+        A["Creating a runtime"]-->B["Generating a chain specification"]-->C["Generate Genesis State & Runtime Artifacts"]
+    end
+
+    subgraph PC["Procure ParaId & Core"]
+        direction LR
+        PARAID["Reserve ParaId"]
+        PARAID-->D["Buy Bulk Coretime"]
+        PARAID-->E["Issue On-Demand Coretime Extrinsic"]
+    end
+
+
+    subgraph DEP["Deploying"]
+        direction LR
+        F["Register artifacts to ParaId"]-->assign["Assign Core"]-->G["Sync collator"]-->H["Begin producing blocks"]
+    end
+
+GA-->PC
+PC-->DEP
+```
+
+<!-- TODO: We should probably have a link to guides or something here, or build some track which has them go from template to core? -->
+
+<!-- TODO: Add a place / page to install the necessary dependencies -->
```

Comment:
We should check where this page will be in the is and migrate/create it

Reply:
It is merged as of now: https://github.com/polkadot-developers/polkadot-docs/pull/24